fetch_controllers/member.go

```diff
@@ -0,0 +1,170 @@
+	err := db.Insert(mem, ct)
+	if err != nil {
+		logger.Error("Failed to save member:", mem.FullName())
+		return err
+	}
+
+	logger.Log("Successfully saved member:", mem.FullName())
+	return nil
+}
+
+func (mfc *MemberFetchController) FetchAllMembers(ft workers.FetchType, congress int) error {
+	urls, err := mfc.InitialFetch(ft, congress)
+	if err != nil {
+		logger.Error("Error: Failed to fetch member urls from propublica with error:", err)
+		return err
+	}
+
+	members := make([]models.Member, 0, len(*urls))
+	for _, url := range *urls {
+		mems, err := mfc.FetchMember(url)
+		if err != nil {
+			logger.Warn("Failed to fetch from", ft, "at url:", url, "error:", err)
+			continue
+		}
+
+		for _, m := range *mems {
+			members = append(members, m)
+			logger.Log("Fetched:", m.FullName(), "from", ft, "for congress:", congress)
+		}
+	}
+
+	if ft == workers.HouseFetch {
+		mfc.Representatives = &members
+	} else {
+		mfc.Senators = &members
+	}
+
+	return nil
+}
+
+func (mfc *MemberFetchController) SaveAllMembers(ft workers.FetchType) error {
+	if ft == workers.HouseFetch && mfc.Representatives == nil {
+		logger.Error("Can't save representatives. No representatives have been saved to the member fetch controller.")
+		return errors.New("No Representatives To Save")
+	}
+
+	if ft == workers.SenateFetch && mfc.Senators == nil {
+		logger.Error("Can't save senators. No senators have been saved to the member fetch controller.")
+		return errors.New("No Senators To Save")
+	}
+
+	var ct db.CollectionType
+	var members *[]models.Member
+	switch ft {
+	case workers.HouseFetch:
+		members = mfc.Representatives
+		ct = db.HouseCollection
+	case workers.SenateFetch:
+		members = mfc.Senators
+		ct = db.SenateCollection
+	}
+
+	if len(*members) == 0 {
+		logger.Warn(
+			"Can't Save member for:",
+			ft,
+			"No member of that type have been saved to the member fetch controller.",
+		)
+		return nil
+	}
+
+	for _, mem := range *members {
+		err := mfc.SaveMember(&mem, ct)
+		if err != nil {
+			logger.Warn("Failed to save member:", mem.FullName(), "to:", ft)
+		}
+	}
+
+	return nil
+}
+
+func (mfc *MemberFetchController) FetchAndSaveMembers(ft workers.FetchType, congress int) error {
+	err := mfc.FetchAllMembers(ft, congress)
+	if err != nil {
+		logger.Error("Failed to fetch and save members with fetch error:", err)
+		return err
+	}
+
+	err = mfc.SaveAllMembers(ft)
+	if err != nil {
+		logger.Error("Failed to fetch and save members with save error:", err)
+		return err
+	}
+
+	return nil
+}
+
+func (mfc *MemberFetchController) FetchAndSaveSenators(congress int) error {
+	return mfc.FetchAndSaveMembers(workers.SenateFetch, congress)
+}
+
+func (mfc *MemberFetchController) FetchAndSaveRepresentatives(congress int) error {
+	return mfc.FetchAndSaveMembers(workers.HouseFetch, congress)
+}
+
+func (mfc *MemberFetchController) FetchAndSaveSenatorsAndRepresentatives(congress int) error {
```

Comment:
I'll add handling to the `octopus.go` file. I don't think it really matters since we're just running a script, but I guess it would be good to be exact.